front/src/components/DataRow/DataRow.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import styles from './DataRow.module.css';
+
+function DataRow(props) {
```

Comment:
Здесь можно сразу props деструктуризировать
```suggestion
function DataRow({ hash, dataObj, columns}) {
```